src/generators/typescript/renderers/EnumRenderer.ts

```diff
@@ -36,13 +36,59 @@ ${this.indent(this.renderBlock(content, 2))}
   runItemPreset(item: any): Promise<string> {
     return this.runPreset('item', { item });
   }
+
+  normalizeKey(value: any): any {
+    let key;
+    switch (typeof value) {
+    case 'bigint':
+    case 'number': {
+      key = `number ${value}`;
+      break;
+    }
+    case 'object': {
+      key = JSON.stringify(value);
+      break;
+    }
+    default: {
+      key = String(value);
+    }
+    }
+    return FormatHelpers.toConstantCase(key);
+  }
+
+  normalizeValue(value: any): any {
+    let normalizedValue;
+    switch (typeof value) {
+    case 'string':
+      normalizedValue = `"${value}"`;
+      break;
+    case 'boolean':
+      normalizedValue = `"${value}"`;
+      break;
+    case 'bigint':
+    case 'number': {
+      normalizedValue = value;
+      break;
+    }
+    case 'object': {
+      normalizedValue = `'${JSON.stringify(value)}'`;
```

Comment:
Should object be stringified or treated as normal JS value (object value)? 🤔

Reply:
Cant be treated as a normal object value, wont allow me to.. So have to do some custom marshalling code to handle this 🙃